cmd/main.go

```diff
@@ -160,6 +163,8 @@ func (f *Aggregator) Aggregate(targets []string, output io.Writer) {
 
 		allFamilies := make(map[string]*io_prometheus_client.MetricFamily)
 
+		var newFamilies *io_prometheus_client.MetricFamily
```

Comment:
I think its ok, because we set it every time for each MetricFamily on line 202 - https://github.com/psvmcc/prometheus-aggregate-exporter/blob/3283056829d94912c737c6e1a8abd705d0561c49/cmd/main.go#L202

Reply:
Ah, so it does. I would still probably move the declaration down to around 197 since it's only used in that block.

Reply:
Done.